Pull Request

```diff
@@ -0,0 +1,6 @@
+I don't know how to make pull request from a repository that has not been forked from the hackathon repository.
+Please guide me.
```

Comment:
You've already forked it.
Now edit the README file and replace the "Sample Project" placeholder with your project's details.